SwanProjects/swanprojects/stacks.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
This is what @peremato and @diocas proposed to change in the last SWAN meeting. As discussed with @omazapa , this could be placed in configuration files living in multiple subfolders, one per stack. Inside each subfolder you would find the configuration for a particular stack, its icon and a script to set the environment for that stack (so that `swan_env` can be made generic). The dialog to create projects would dynamically find the information about the available stacks and display it.

Does this sound good?

Reply:
I made some comments but I believe they are all aligned with this one. If not, this takes precedence :)
But my problem is that this assumes a similar structure (stack, env), while this is not always true (i.e conda).

Reply:
done!